migrations/versions/201608161607_1b66d038da46_add_render_mode_column_to_sub_.py

```diff
@@ -0,0 +1,48 @@
+"""Add render_mode column to (sub-)contribution descriptions
+
+Revision ID: 1b66d038da46
+Revises: 8763fe20f41
+Create Date: 2016-08-16 16:07:25.177573
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from indico.core.db.sqlalchemy import PyIntEnum
+from indico.core.db.sqlalchemy.descriptions import RenderMode
+
+
+# revision identifiers, used by Alembic.
+revision = '1b66d038da46'
+down_revision = '8763fe20f41'
+
+
+def upgrade():
+    markdown_mode = str(RenderMode.markdown)
+    op.add_column('contributions',
+                  sa.Column('render_mode', PyIntEnum(RenderMode), server_default=markdown_mode, nullable=True),
+                  schema='events')
+    op.add_column('subcontributions',
+                  sa.Column('render_mode', PyIntEnum(RenderMode), server_default=markdown_mode, nullable=True),
+                  schema='events')
+
+    # set 'legacy' (sub-)contributions to 'html'
+    op.execute("""UPDATE events.contributions
+                    SET render_mode = %d
+                    WHERE EXISTS (
+                        SELECT 1 FROM events.legacy_contribution_id_map
+                                 WHERE legacy_contribution_id_map.contribution_id = contributions.id)
+               """ % (RenderMode.html))
+    op.execute("""UPDATE events.subcontributions
+                    SET render_mode = %d
+                    WHERE EXISTS (
+                        SELECT 1 FROM events.legacy_subcontribution_id_map
+                                 WHERE legacy_subcontribution_id_map.subcontribution_id = subcontributions.id)
+               """ % (RenderMode.html))
```

Comment:
Those queries are not super fast. Not sure if there's a more efficient way of phrasing them.